src/spectrecoff/Output.fs

```diff
@@ -43,15 +43,15 @@ let rec private joinSeparatedBy (separator: string) (strings: string list) =
 let private stringifyDecorations (decorations: Decoration list) =
     decorations |> List.map (fun decoration -> decoration.ToString())
 
-let private stringify foregroundColorOption backgroundColorOption decorations =
+let private stringify (foregroundColorOption: Color option) (backgroundColorOption: Color option) decorations =
```

Comment:
I checked out the branch, and it works fine for both rgb and named values! 
![image](https://github.com/EluciusFTW/SpectreCoff/assets/10219667/13f14446-39c1-4cbd-b77f-046a2e832e5f)

Would you mind to altering the example in the cli similar to the screenshot so both cases are 'tested'?
(If you run the command, don't be bothered by an error further down [Could not find color or style 'documentation!'], I just realized in a recent refactoring that I misordered teh argumetns in the `LinkWithLabel` type, will post a fix for that later today/tomorrow).